tdrs-backend/gunicorn_start.sh

```diff
@@ -12,16 +12,17 @@ else
 fi
 
 # Collect static files. This is needed for swagger to work in local environment
-if [[ $DISABLE_COLLECTSTATIC ]]; then 
+if [[ $DISABLE_COLLECTSTATIC ]]; then
     echo "DISABLE_COLLECTSTATIC is set to true, skipping collectstatic"
 else
     echo "Collecting static files"
     python manage.py collectstatic --noinput
 fi
 
 
-celery -A tdpservice.settings worker -c 1 &
+celery -A tdpservice.settings worker --loglevel=WARNING --concurrency=1 -n worker1@%h &
```

Comment:
What do the special symbols ending of `worker1@%h` indicate?

Reply:
It is a syntax that celery recommends to associate each worker with the host it is running on. In the future if/when we add more workers it will be a little easier to delineate workers having issues. Especially if we allow workers to span multiple hosts.

Reply:
could you add the link as a docstring for future ref?